finality-aleph/src/data_io/status_provider.rs

```diff
@@ -413,5 +413,15 @@ mod tests {
             &fresh_proposal,
             Finalize(fresh_proposal.blocks_from_num(0).collect()),
         );
+
+        // Long proposals should finalize the appropriate suffix.
+        let long_proposal = proposal_from_blocks(blocks[0..6].to_vec());
```

Comment:
Hmm, this seems to be the only line which mentions `long_proposal` so I'm a little suspicious :P.

Reply:
Whoopsie, should be better now.